clients/ui/frontend/src/app/pages/modelCatalog/components/ValidatedModelCard.tsx

```diff
@@ -0,0 +1,247 @@
+
+  const metrics = extractValidatedModelMetrics(performanceMetrics, accuracyMetrics);
+
+  return (
+    <Card
+      isFullHeight
+      data-testid="validated-model-catalog-card"
+      key={`${model.name}/${model.source_id}`}
+    >
+      <CardHeader>
+        <CardTitle>
+          <Flex alignItems={{ default: 'alignItemsCenter' }} style={{ paddingLeft: '8px' }}>
+            {model.logo ? (
+              <img
+                src={model.logo}
+                alt="model logo"
+                style={{ height: '56px', width: '56px', marginRight: '8px' }}
+              />
+            ) : (
+              <Skeleton
+                shape="square"
+                width="56px"
+                height="56px"
+                screenreaderText="Brand image loading"
+                style={{ marginRight: '8px' }}
+              />
+            )}
+            <FlexItem align={{ default: 'alignRight' }}>
+              <Badge className="pf-m-purple">Validated</Badge>
+            </FlexItem>
+          </Flex>
+          <div style={{ marginTop: '8px' }}>
+            <Flex alignItems={{ default: 'alignItemsCenter' }} gap={{ default: 'gapSm' }}>
+              <div
+                data-testid="validated-model-status-indicator"
+                className={text.textColorStatusSuccess}
+                style={{
+                  width: '8px',
+                  height: '8px',
+                  backgroundColor: 'var(--pf-t--global--palette--green-400)',
+                  borderRadius: '50%',
+                }}
+              />
+              <Content component={ContentVariants.small} className={text.textColorStatusSuccess}>
+                Success
+              </Content>
+            </Flex>
+            <Link to={catalogModelDetailsFromModel(model.name, source?.id)}>
+              <Button
+                data-testid="model-catalog-detail-link"
+                variant="link"
+                tabIndex={-1}
+                isInline
+                style={{ fontSize: '1.25rem', fontWeight: 'bold' }}
+              >
+                {truncate ? (
+                  <span>{getModelName(model.name)}</span>
+                ) : (
+                  <span>{getModelName(model.name)}</span>
+                )}
+              </Button>
+            </Link>
+          </div>
+        </CardTitle>
+      </CardHeader>
+      <CardBody>
+        <div>
+          <div style={{ paddingLeft: '8px', marginBottom: '16px' }}>
+            <Flex
+              alignItems={{ default: 'alignItemsCenter' }}
+              justifyContent={{ default: 'justifyContentSpaceBetween' }}
+            >
+              <Flex
+                alignItems={{ default: 'alignItemsCenter' }}
+                gap={{ default: 'gapSm' }}
+                wrap="nowrap"
+              >
+                <ChartBarIcon />
+                <span
+                  data-testid="validated-model-accuracy"
+                  style={{ fontSize: '1.25rem', fontWeight: 'bold' }}
```

Comment:
The fontSize is unnecessary and makes the text too big, it seems to me to be the same size as other text on the card but just bold. You can make the text bold with `className="pf-v6-u-font-weight-bold"` instead of an inline style. [PF Text utility classes](https://www.patternfly.org/utility-classes/text)